src/sdk-reference/bulk/import.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: full.html
+algolia: true
+title: import
+description: Performs a bulk import on a collection
+order: 200
+---
+
+# import(data, [options])
+
+The bulk import allows to save a list of documents into a specific collection (belonging to a specific index). If a subset of the documents fails to save, a PartialError is triggered. The `data` argument passed to the method must specify the set of documents to import and must satisfy the [Elasticsearch Bulk API](https://www.elastic.co/guide/en/elasticsearch/reference/5.5/docs-bulk.html).
+
+### The Elasticsearch Bulk API in brief
+
+This API takes a JSON array containing a list of JSON objects working in pairs. In each pair, the first object specifies the action to perform (the most common is `create`) and the second specifies the document itself, like in the example below:
+
+```javascript
+[
+    // The action object
+    { create: { _id: 'id', _index: 'index', _type: 'collection'}}
+    // The document object
+    { myField: 'myValue', myOtherField: 'myOtherValue' }
+
+    // Another action object
+    { create: { _id: 'another-id', _index: 'index', _type: 'collection'}}
+    // Another document object
+    { myField: 'anotherValue', myOtherField: 'yetAnotherValue' }
+]
+```
+
+Note that the action object always has an attribute whose key specifies the action to take and whose value is an object specifying the location of ano object in the database (the `_index`, `_type` and `_id` tuple). Also note that, in Elasticsearch, the field `_type` correspond to `collection` in Kuzzle.
+
+Possible actions are `create`, `index`, `update`, `delete`.
+
+Learn more at https://www.elastic.co/guide/en/elasticsearch/reference/5.5/docs-bulk.html
+
+## Usage
+
+[code-example=import]
+
+## Options
+
+Query options.
+
+| Option    | Type   | Description                                         | Default |
+| --------- | ------ | --------------------------------------------------- | ------- |
+| `data`    | Object | The list of documents to be added to the collection | {}      |
```

Comment:
first column data are not displayed in the preview ???

Reply:
Yep, this is a bug @Njuelle is working on.